src/Microsoft.Health.Dicom.Web/appsettings.json

```diff
@@ -21,7 +21,7 @@
                 "BatchSize": 10
             },
             "StoreServiceSettings": {
-                "MaxAllowedDicomFileSize": 2147483648
+                "MaxAllowedDicomFileSize": 4294967295
```

Comment:
No body is asking for 4GB, so lets leave it at 2GB for now.

Reply:
hi Smitha, just clarify https://microsofthealth.visualstudio.com/Health/_workitems/edit/75662, so customer ask for >2GB or <=2GB?